enterprise-cfengine-guide/hosts-health.markdown

```diff
@@ -21,11 +21,14 @@ Here you will find extensive information on single hosts that CFEngine detects a
 
 ## Host Health ##
 
-![Hosts](Mission-Portal-Hosts-1.png)
+![Hosts](Mission-portal-health-dignostics-header.png)
 
 You can get quick access to the health of hosts, including direct links to reports, from the Health drop down at the top of every Enterprise UI screen. Hosts are listed as unhealthy if:
 
 * the hub was not able to connect to and collect data from the host within a set time interval (unreachable host). The time interval can be set in the Mission Portal settings.
 * the policy did not get executed for the last three runs. This could be caused by `cf-execd` not running on the host (scheduling deviation) or an error in policy that stops its execution. The hub is still able to contact the host, but it will return stale data because of this deviation.
+* two or more hosts use the same key. This is detected if the IP address tied to a CFEngine key has changed in the last three scheduled runs. The number of scheduled runs that cause the unhealthy status is configurable in [settings][Settings#Preferences].
+* reports have recently been collected, but cf-agent has not recently “gracefully” run. “Recently” is defined by the configured run-interval of their cf-agent.
```

Comment:
the setting is named "Number of samples used to identify a duplicate identity" right?

Reply:
Didn't @roschern want "gracefully" removed or replaced here? Maybe a comment in e-mail? I can't find it now.

Reply:
@craigcomstock  Ah I've forgotten about gracefully. One moment

>the setting is named "Number of samples used to identify a duplicate identity" right? 

right

Reply:
done